man/adoc/bpftrace.adoc

```diff
@@ -2502,12 +2502,25 @@ Count how often this function is called.
 
 Using `@=count()` is conceptually similar to `@++`.
 The difference is that the `count()` function uses a map type optimized for
-writing (PER_CPU), increasing performance and correctness. However, sync reads
+performance and correctness using cheap, thread-safe writes (PER_CPU). However, sync reads
 can be expensive as bpftrace needs to iterate over all the cpus to collect and
 sum these values.
-Note: In contrast to hash maps (e.g. `@++`), multiple writers to a shared
-global var might lose counts as bpftrace doesn't update them atomically.
 
+Note: This differs from "raw" writes (e.g. `@++`) where multiple writers to a
+shared location might lose updates, as bpftrace does not generate any implicit
```

Comment:
```suggestion
shared map might lose updates, as bpftrace does not generate any implicit
```

Reply:
I wasn't sure about this. "Shared map" may not be the clearest way to express what we want. For example, what about shared writes to `@m[k]`? It's not exactly a map, it's an entry in the map. So I thought "location" is more appropriate. WDYT?

Reply:
Ehh.. it's still a value within a **map** but I understand what you're saying. I won't block. Let's see if users get confused.